doc/sphinx/source/almaif.rst

```diff
@@ -249,6 +249,29 @@ Example usage of the mode can be found in examples/accel/CMakelists.txt, which
 generates standalone tests using both ttasim and RTL simulator (ghdl) to run the
 example0 kernel on various TTA configurations.
 
+
+Using bitstream database
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can use the AlmaIF-driver with the cross-vendor bitstream databases generated
+with the `AFOCL-project <http://github.com/cpc/AFOCL>`_.
+That project generates a directory-based database with a json-based metadata.
+The database contains the bitstreams and firmware-files necessary to implement
+the set of built-in kernels defined in the json-file.
+
+To use AFOCL-databases in PoCL, it is enough to point the Almaif-driver to the database
+with the env variable::
+
+  POCL_DEVICES=almaif POCL_ALMAIF0_PARAMETERS=0xF,<path/to/afocl-db> ./accel_example
+
+At the moment, the public AlmaIF-driver and AFOCL include support only for
+Xilinx Alveo U280 device, but adding support for other Alveo devices should be easy.
+In AFOCL publication the methodology was also demonstrated with Intel Arria 10,
+but the code for that is not yet upstreamed.
+For more information about the bitstream database,
+see our AFOCL-publication. (To be presented at NorCAS conference on 31.10.2023).
```

Comment:
Perhaps add a preprint to Arxiv or ResearchGate and link to it? At least add full citation info here so it's easier to search for it later.

Reply:
added a citation there. Not bothering with the pre-print since the conference is coming up so soon